packages/babel-plugin-transform-typescript/src/namespace.ts

```diff
@@ -69,11 +69,8 @@ function handleVariableDeclaration(
   const { declarations } = node;
   if (
     declarations.every(
-      (
-        declarator,
-      ): declarator is t.VariableDeclarator & {
-        id: t.Identifier;
-      } => t.isIdentifier(declarator.id),
+      (declarator): declarator is t.VariableDeclarator & { id: t.Identifier } =>
+        t.isIdentifier(declarator.id),
```

Comment:
Oh I tried doing this locally, but for some reason (maybe VS code was using the prev prettier version?) it didn't work :grimacing: